x-pack/plugins/secops/public/containers/who_am_i/index.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { Query } from 'react-apollo';
+import { WhoAmIQuery } from '../../../common/graphql/types';
+
+import { whoAmIQuery } from './who_am_I.gql_query';
+
+interface WHoAmIArgs {
```

Comment:
Capital `WH` ?, Think you mean `WhoAmIArgs`

Reply:
👍 got it now